crates/viewer/re_viewer/src/app.rs

```diff
@@ -232,11 +227,10 @@ pub struct App {
 
     reflection: re_types_core::reflection::Reflection,
 
-    /// Interaction between JS and timeline.
+    /// Interaction between JS and the viewer.
```

Comment:
same comment fix as above applies here, talking about JS only here is _kinda_ wrong and definitely confusing :)